core/variant/variant_utility.cpp

```diff
@@ -1228,6 +1143,15 @@ bool VariantUtilityFunctions::is_same(const Variant &p_a, const Variant &p_b) {
 	return p_a.identity_compare(p_b);
 }
 
+String VariantUtilityFunctions::join(const Variant **p_args, int p_arg_count, Callable::CallError &r_error) {
+	String s;
+	for (int i = 0; i < p_arg_count; i++) {
+		String os = p_args[i]->operator String();
+		s += os;
+	}
+	return s;
```

Comment:
```suggestion
	r_error.error = Callable::CallError::CALL_OK;
	return s;
```
If you are to bind this you need to return an ok as well

Reply:
@AThousandShips I believe you are correct that we don't really need a binding, so I have just removed the binding and the `r_error` argument.